### ISSUE TITLE
restart of apache required after upgrade

### ISSUE THREAD

**Issue by nevart** [systemli/ansible-role-nextcloud#6]
After upgrading to `18.0.3`, the following message was displayed:

```
apps directory not found! Please put the Nextcloud apps folder in the Nextcloud folder or the folder above. You can also configure the location in the config.php file.
```

An Apache restart was required in order to solve this.

Due to my limited ansible knowledge I'm not really sure what the proper fix would look like, but I'm all ears for any tips and tricks. Should this just be the last step in the playbook?

**Comment by t2d** [comment]
Yes, that would be great. Handlers are the required action: https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html#handlers-running-operations-on-change

**Comment by t2d** [comment]
Fix by de23bc42f6c1178e8c18c93aff5b11e40ded721a
